node/src/Consumer.ts

```
@@ -789,9 +790,24 @@ export class Consumer<ConsumerAppData extends AppData = AppData>
 			throw new TypeError('every type must be a string');
 		}
 
+		// Convert event types.
+		const fbsEventTypes: FbsConsumer.TraceEventType[] = [];
+
+		for (const eventType of types)
+		{
+			try
+			{
+				fbsEventTypes.push(consumerTraceEventTypeToFbs(eventType));
+			}
+			catch (error)
+			{
+				// Ignore invalid event types.
```

Same as in producer PR, I think error should be printed and not swallowed in such cases, might be hard to debug otherwise.